src/reducers/patients.js

```diff
@@ -0,0 +1,13 @@
+import { FETCH_PATIENTS_FULFILLED } from '../actions/types';
+
+export default function(state = {patientEntries: [], meta:{}}, action)  {
```

Comment:
Change `patientEntries` to `patients` for consistency. Remove `meta` as it is not needed here.